spec/system/organization_system_spec.rb

```diff
@@ -108,6 +110,16 @@
         expect(page).to_not have_content('Media Information')
         expect(@organization.reload.partner_form_fields).to eq([])
       end
+
+      it 'can change the allowed request types and send an error if it results in a partner not having request capability' do
+        partner = @organization.partners.first.profile
+        partner.update(enable_child_based_requests: false, enable_individual_requests: false)
+
+        choose('organization[enable_quantity_based_requests]', option: false)
+        click_on "Save"
+
+        expect(page).to have_content("Please update DEFAULT PARTNER, so that they could make requests if you completed this change, then try again. Thank you.")
+      end
```

Comment:
We shouldn't need a system spec for this since there's no interactivity on the frontend. The service spec should be good enough.